src/include/miopen/execution_context.hpp

```diff
@@ -100,12 +101,11 @@ struct ExecutionContext
         // clang-format off
         return GetUserDbPath()
 #if MIOPEN_ENABLE_SQLITE
-             + "/miopen.udb";
+             + "miopen_" + std::string{MIOPEN_PERFDB_SCHEMA_VER} + ".udb";
 #else
-             + "/"
              + GetStream().GetDbBasename()
              + "."
-             + GetUserDbSuffix()
+             + std::string{MIOPEN_PERFDB_SCHEMA_VER}
```

Comment:
Text-based perf-db does not need this, could you please revert.

Reply:
oops, `GetUserDbSuffix()` is removed, looks like that this change was made by accident.

Reply:
No, this was requested by @pfultz2.

Reply:
@JehandadKhan @pfultz2 Hmm. This specific change is wrong because is removes backend name and miopen version from user-find-db name and adds SQL db schema version that is not relevant.

There is RFC ticket related to find-db naming, https://github.com/AMDComputeLibraries/MLOpen/issues/1139. Let's discuss & agree there prior making changes.

Reply:
this line is for the text version of user perf db, which is not backend dependent ?

Reply:
@JehandadKhan @pfultz2 Oops, sorry, I've mixed up different databases ;( Brand new comment:

This change is incorrect because is removes miopen version from the text-based user-perf-db. Also, SQL db schema version is not relevant here.

I do not have specific ticket for the _perf-db naming_, so let's discuss right here.
- Library version is necessary; this is obvious.
- Text-based databases do not change format (or backward compatible at least, -- this is a requirement).

The change also removes backend name, but this question is more complex.
- Related discussion/explanation can be found around [this comment](https://github.com/AMDComputeLibraries/MLOpen/issues/1419#issuecomment-545114745).
- In short, we can live with backend-agnostic system perf-db, but we still need backend-specific user layer ensures correct work in corner cases.
- These corner cases may happen when the user wants _two installations with different backends on the same system_. The one may think that such a case is purely theoretical, however developers can switch backends easily (and I do that often).
- I think that _if we do not want to support such setups, then backend name can be removed from the user-perf-db name._

Reply:
I have reverted the change, since its not relevant to SQLite Perf Db

Reply:
Resolved. Please do not click `[Resolve conversation]` to keep this visible for a while.